authors/apps/articles/migrations/0001_initial.py

```diff
@@ -1,5 +1,6 @@
-# Generated by Django 2.1.5 on 2019-01-21 08:50
+# Generated by Django 2.1.5 on 2019-01-22 10:08
```

Comment:
@calvinpete the initial migration file should only contain @mariamiah chnages. Please rebase to reset this initial commit, delete the migrations that you created and regenerate migrations. we could have a sync to help you with this.